js/README.md

```diff
@@ -0,0 +1,193 @@
+# How to run
```

Comment:
doc-team proof read?

Reply:
Not yet, as stated in PR's TODO it has to be adjusted a little, but I'd rather do it after CR of rest in case of you find some glaring error. :)